src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -24,15 +25,16 @@ const useSaveAnonChatHook = () => {
     return channelListSchema;
   };
 
-  const helperGoToAnonymousChat = async (object: InitAnonymousChatData) => {
+  const helperGoToAnonymousChat = async (object: InitAnonymousChatData, from: string) => {
     const channelListSchema = await helperFindChatById(object);
-    goToChatScreen(channelListSchema);
+    goToChatScreen(channelListSchema, from);
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `helperGoToAnonymousChat` now accepts an additional parameter `from`. Make sure to update all calls to this function to include this new argument. Also, ensure that the `goToChatScreen` function can handle this new parameter.

```diff
- const helperGoToAnonymousChat = async (object: InitAnonymousChatData) => {
-    const channelListSchema = await helperFindChatById(object);
-    goToChatScreen(channelListSchema);
- };
+ const helperGoToAnonymousChat = async (object: InitAnonymousChatData, from: string) => {
+    const channelListSchema = await helperFindChatById(object);
+    goToChatScreen(channelListSchema, from);
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->